README.md

```diff
@@ -37,24 +37,36 @@ Will result into this HTML (open graph metas are using property automatically) :
 
 Title & metas are set only on `didTransition` in case the route would be canceled.
 
+**Note**: og tags will not correctly work without using FastBoot or some sort of server side rendering.
```

Comment:
I would add a link to Fastboot & say Open Graph (og) to be clear.

```md
**Note**: Open Graph (og) tags will not correctly work without using [FastBoot](https://www.ember-fastboot.com/) or some sort of server side rendering.
```

I can do it after merging